model_evaluation/jigsaw_evaluation_pipeline.ipynb

```diff
@@ -210,31 +206,111 @@
+    "    'other_sexual_orientation': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'christian': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'jewish': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'muslim': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'hindu': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'buddhist': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'atheist': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'other_religion': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'black': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'white': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'asian': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'latino': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'other_race_or_ethnicity': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'physical_disability': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'intellectual_or_learning_disability': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'psychiatric_or_mental_illness': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "    'other_disability': tf.FixedLenFeature([], dtype=tf.float32, default_value=-1.),\n",
+    "}\n",
+    "\n",
+    "identity_terms_civil = ['male', 'female', 'transgender', 'other_gender', 'heterosexual', \n",
+    "                        'homosexual_gay_or_lesbian', 'bisexual', 'other_sexual_orientation',\n",
+    "                        'christian', 'jewish', 'muslim', 'hindu', 'buddhist', 'atheist',\n",
+    "                        'other_religion', 'black', 'white', 'asian', 'latino', 'other_race_or_ethnicity',\n",
+    "                        'physical_disability', 'intellectual_or_learning_disability',\n",
+    "                        'psychiatric_or_mental_illness', 'other_disability'\n",
+    "                       ]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def input_fn_performance_civil(max_n_examples=None, random_filter_keep_rate=1.0):\n",
+    "    civil_df_raw = utils_tfrecords.decode_tf_records_to_pandas(\n",
+    "        civil_comments_spec,\n",
+    "        CIVIL_COMMENTS_PATH,\n",
+    "        max_n_examples=max_n_examples,\n",
+    "        random_filter_keep_rate=random_filter_keep_rate,\n",
+    "    )\n",
+    "    civil_df_raw[TEXT_FEATURE_NAME] = list(map(tokenizer, civil_df_raw[TEXT_FEATURE_NAME]))\n",
+    "    civil_df_raw['toxicity'] = list(map(lambda x :bool(round(x)), list(civil_df_raw['toxicity'])))\n",
```

Comment:
I think the spacing after lambda x is wrong? Shouldn't it be 'lambda x: bool....'?

Reply:
Thanks for spotting it